pippo-core/src/main/java/ro/fortsoft/pippo/core/Response.java

```diff
@@ -62,12 +62,14 @@ public String getContentType() {
     }
 
     public Response contentType(String contentType) {
+        checkCommitted();
```

Comment:
I want to clarify one point. This is not necessary in terms of the servlet theory and it is only a protection for developers (means that theoretically you can change the content type after commit but it's a non sense). Is it right what I say?